tests/infra/ccf.py

```diff
@@ -602,6 +592,34 @@ def set_lua_app(self, node, app_script):
         )
         self.vote_using_majority(node, result[1]["id"])
 
+    def accept_recovery(self, node, sealed_secrets):
+        result = self.propose(
+            1, node, None, None, "accept_recovery", f"--sealed-secrets={sealed_secrets}"
+        )
+        self.vote_using_majority(node, result[1]["id"])
+
+    def wait_for_all_nodes_to_be_trusted(self, timeout=3):
+        primary, term = self.find_primary()
+        for n in self.nodes:
+            self._wait_for_node_to_exist_in_store(
+                primary, n.node_id, NodeStatus.TRUSTED
+            )
+
+    def _wait_for_node_to_exist_in_store(
+        self, remote_node, node_id, node_status=None, timeout=3
```

Comment:
I think we upped this `timeout` up to 30

Reply:
Good spot. `20` actually :)

Reply:
That function was actually duplicated in `ccf.py`!